### PR TITLE
Adds a medal for getting to the end of the secret tile dialog.

### PR DESCRIPTION
**Pull request by Shadowlight213**
:cl:
add: You now can get a medal for wasting hours talking to the secret debug tile.
fix: Fixed runtime for the tile when poly's speech file doesn't exist.
/:cl:

Reaching the end of the secret tile's dialog now gives you a medal along with the trophy so you can have that sense of pride and accomplishment that you wasted so much time on something so stupid.

Also fixes a runtime when the poly file doesn't exist.

image

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/awaymissions/super_secret_room.dm [hunk 1]
```diff
@@ -45,7 +47,7 @@
 			SpeakPeace(list("Alright maybe that's <b>too</b> boring.", "I can't keep manually typing these lines out though.", "It's hard to explain but the code structure I'm using is kind of terrible."))
 		if(10)
 			SpeakPeace(list("Oh I have an idea!", "Lets outsource this endless banter to Poly!", "Then you'll be able to keep listening to this without getting bored!"))
-			if(isnull(shenanigans))
+			if(isnull(shenanigans) || !shenanigans.len)
 				shenanigans = list("Except the poly file is missing...")
 		if(11 to 14, 16 to 50, 52 to 99, 103 to 107, 109 to 203, 205 to 249, 252 to 665, 667 to 999, 1001 to 5642)
 			SpeakPeace(list(pick(shenanigans),pick(shenanigans),pick(shenanigans)))
```
Comment (lordpidey) [inline code/modules/awaymissions/super_secret_room.dm]: While we're editing this file, could you make the line RIGHT below this one more sane?
`if (times_spoken_to % 10 == 0)` or something like that?

Comment (Shadowlight213) [reply code/modules/awaymissions/super_secret_room.dm]: Not sure what you mean here

Comment (Jordie0608) [reply code/modules/awaymissions/super_secret_room.dm]: To change https://github.com/tgstation/tgstation/blob/5a5f9c289110fb52077dcafe939fe7d15763d82d/code/modules/awaymissions/super_secret_room.dm#L52
to `if(times_spoken_to % 10 == 0)` (which is actually slightly different behavior but that's fine)

Comment (optimumtact) [reply code/modules/awaymissions/super_secret_room.dm]: yo

**Pull request merged**